### ISSUE TITLE
Scales from an animated GIF are not animated

### ISSUE THREAD

**Issue by hvelarde** [plone/Products.CMFPlone#1871]
Here you have an animated GIF:

![timeline-2](https://cloud.githubusercontent.com/assets/341393/21076036/7da3037c-bf08-11e6-8acb-334fc4e94192.gif)

Here you have one of the resulting scales:

![2c780b04-5311-4577-9425-d3bdbe92b8d0](https://cloud.githubusercontent.com/assets/341393/21076038/98d701a2-bf08-11e6-9bb4-9188aa4ba89e.jpeg)

**Comment by jensens** [comment]
Indeed, this was never implemented in ``plone.scale``. Major reason: PIL does not support it out of the box. Not that it would be impossible, but nobody implemented it yet.

**Comment by tomgross** [comment]
There was a disscussion on this some time ago: http://stackoverflow.com/questions/9128811/how-do-you-scale-an-animated-gif-image-in-pil-and-preserve-the-animation#13024524

**Comment by hvelarde** [comment]
seems PIL supports animated GIF since version 3.0.0:

https://github.com/python-pillow/Pillow/pull/1384

**Comment by jensens** [comment]
Cool, but it seems it needs some extra love to make it work in plone.scale. Any takers?

**Comment by jensens** [comment]
Checked on Plone 6: GIFs are scaled to PNGs and so are not animated.

**Comment by jensens** [comment]
wrong, fixed in 6, broken in 5.2
I do not think this will be fixed in 5.2 and close this.
